src/site/apt/eclipse/google_plugin.apt.vm

```diff
@@ -144,7 +144,7 @@ Using the Google Eclipse Plugin
   configured (*) as a GWT module project, changes to java sources will be available in DevMode with a simple
   refresh with no requirement to repackage the modules. 
   
-[../images/eclipse/gwt-maven-logo.png] m2eclipse detecting the gwt module as project reference.
+[../images/gwt-maven-logo.png] m2eclipse detecting the gwt module as project reference.
```

Comment:
Now that both GWT and Maven have new logos, I think that one needs to be updated ;-)